GothicVRProject/GothicVR
# Feature/barrier

# To test
1. Make sure that Barrier Flag is set
2. Start World and check if barrier exists
3. Compare our barrier with original game if it's the same waviness



# Testing
